<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BacklogWidget</class>
 <widget class="QDialog" name="BacklogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Backlog</string>
  </property>
  <layout class="QHBoxLayout" name="BacklogWidgetHorizontalLayout">
   <item>
    <widget class="QGroupBox" name="epicsGroupBox">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Эпики</string>
     </property>
     <layout class="QVBoxLayout" name="epicsGroupBoxVerticalLayout">
      <item>
       <widget class="QTableWidget" name="epicsTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="storiesGroupBox">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Стори</string>
     </property>
     <layout class="QVBoxLayout" name="storiesGroupBoxVerticalLayout">
      <item>
       <widget class="QTableWidget" name="storiesTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="tasksGroupBox">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Таски</string>
     </property>
     <layout class="QVBoxLayout" name="tasksGroupBoxVerticalLayout">
      <item>
       <widget class="QTableWidget" name="tasksTableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
